docs/components/modeler/web-modeler/connectors/available-connectors/sendgrid.md

```diff
@@ -0,0 +1,195 @@
+
+## SendGrid connector
+
+The SendGrid connector comes with two options:
+
+1. **SendGrid Email Connector** allows sending simple emails (i.e. text/plain, text/html).
+2. **SendGrid Email Template Connector** supports [SendGrid Dynamic Templates](https://sendgrid.com/solutions/email-api/dynamic-email-templates/).
+
+### SendGrid Email Connector
+
+#### Create a SendGrid Email Connector Task
+
+To use this connector, follow the steps below:
+
+1. Open Web Modeler and create a new BPMN diagram.
+2. Add a **Task** and an **EndEvent**.
+3. Click on the **Task**, click on the wrench icon, and select **SendGrid Email Connector**. You can also use the search field and type `Sendgrid`.
+
+![create sendgrid connector wrench](../img/connectors-sendgrid-create-task-wrench.png)
+
+Alternatively, you can directly create a **SendGrid Email Connector** by using the append menu:
+
+![create sendgrid connector append](../img/connectors-create-task-append.png)
+
+#### Make your SendGrid Email Connector executable
+
+To make the **SendGrid Email Connector** executable, you need to fill out all the mandatory fields highlighted in red in the properties panel:
+
+![sendgrid email connector red properties](../img/connectors-sendgrid-email-red-properties.png)
+
+1. Set **SendGrid API Key** to `secrets.SEND_GRID_API_KEY`.
+2. Set **Sender Name** to `Jane Doe` (or the [sender identity](#create-a-sender-identity) you configured above).
+3. Set **Sender Email** to `jane-doe@camunda.com` (or the [sender identity](#create-a-sender-identity) you configured above).
+4. Set **Receiver Name** to `Your Name`.
+5. Set **Receiver Email** to `Your email address`.
+6. Set **Email Content Subject**.
+7. Leave **Content Type** to **text/plain** (or alternatively to **text/html** if you intend to provide an HTML body to your email).
+8. Provide a text (or HTML) **Body** for your email.
+
+![sendgrid email connector complete properties](../img/connectors-sendgrid-email-complete-properties.png)
+
+:::note
+Now you can simply [deploy and start a new instance](../../save-and-deploy.md) of your process.
+The connector execution logic will be available for you at runtime.
+:::
+
+### SendGrid Email Template Connector
+
+Send an email via SendGrid Dynamic Template and use the [Handlebars templating language](https://handlebarsjs.com/) to pass dynamic values to your connector.
+
+#### Configure a Dynamic Template
+
+1. Open the [Dynamic Transactional Templates page](https://sendgrid.com/dynamic_templates) and click **Create Template**.
+2. Add a unique template name and click **Save**.
+3. To begin editing your new template, click **Add Version**.
+4. Select an editor and click **Continue**.
+5. Design your template. For more information on using Handlebars, click [here](https://docs.sendgrid.com/for-developers/sending-email/using-handlebars).
+
+In our example template we will use the following subject and body:
+
+```text
```

Comment:
Did we want/need to update this example as Camunda Platform 8 instead of Camunda Cloud?

Reply:
I updated all screenshots and texts